internal/provider/provider.go

```diff
@@ -49,12 +50,16 @@ func (p *hetznerDNSProvider) Schema(_ context.Context, _ provider.SchemaRequest,
 				Sensitive:   true,
 			},
 			"max_retries": schema.Int64Attribute{
-				Description: "The maximum number of retries to perform when an API request fails. Default: 1",
+				Description: "The maximum number of retries to perform when an API request fails. You can pass it using the env variable `HETZNER_DNS_MAX_RETRIES` as well. Default: 1",
 				Optional:    true,
 				Validators: []validator.Int64{
 					int64validator.AtLeast(0),
 				},
 			},
+			"enable_txt_formatter": schema.BoolAttribute{
+				Description: "Toggles the automatic formatter for TXT record values. You can pass it using the env variable `HETZNER_DNS_TXT_FORMATTER_ENABLE` as well. Default: true",
```

Comment:
```suggestion
				Description: "Toggles the automatic formatter for TXT record values. Values greater than 255 bytes needs to be chunked and quotes separetly. You can pass it using the env variable `HETZNER_DNS_TXT_FORMATTER_ENABLE` as well. Default: true",
```

We should have an consistency at the env variables, like:  `HETZNER_DNS_<attr>`

instead `HETZNER_DNS_TXT_FORMATTER_ENABLE`, name it `HETZNER_DNS_ENABLE_TXT_FORMATTER`